apps/client/src/config/autofill.js

```diff
@@ -55,7 +55,7 @@ export const autofillResolvers = {
  * Map from autofill-resolver key to the data-need it has.
  */
 export const autofillMap = {
-  monumentList: "address",
-  monumentBoolean: "address",
-  cityScape: "address",
+  monumentList: "wrapper",
+  monumentBoolean: "wrapper",
+  cityScape: "wrapper",
```

Comment:
To fulfill the data need for autofill questions we need to render additional components on the wrapper page. For the `outcomes` sttr-test we don't have any data needs so no autofill and no location/address component. In that case we don't want to render the address form. So this autofillMap is still useful in it's original implementation, please don't change this but use the configuration instead to render additional components on the wrapper page.

Reply:
Let's discuss this, it's wrapper for now, otherwise the page is broken. 
But lets make it part of the refactor.